RevenueCatUI/Templates/MultiPackageTemplate.swift

```diff
@@ -0,0 +1,241 @@
+                } label: {
+                    self.packageButton(package, selected: self.selectedPackage === package.content)
+                }
+                .buttonStyle(PackageButtonStyle())
+            }
+        }
+        .padding(.bottom)
+    }
+
+    @ViewBuilder
+    private func packageButton(_ package: TemplateViewConfiguration.Package, selected: Bool) -> some View {
+        let alignment: Alignment = .leading
+
+        VStack(alignment: alignment.horizontal, spacing: 5) {
+            HStack {
+                Image(systemName: "checkmark.circle.fill")
+                    .foregroundColor(
+                        selected
+                        ? self.configuration.colors.callToActionBackgroundColor
+                        : .gray
+                    )
+
+                Text(package.content.productName)
+            }
+            .foregroundColor(self.configuration.colors.callToActionBackgroundColor)
+
+            IntroEligibilityStateView(
+                textWithNoIntroOffer: package.localization.offerDetails,
+                textWithIntroOffer: package.localization.offerDetailsWithIntroOffer,
+                introEligibility: self.introEligibility[package.content],
+                foregroundColor: selected
+                    ? .white
+                    : .black,
+                alignment: alignment
+            )
+            .fixedSize(horizontal: false, vertical: true)
+            .font(.body)
+        }
+        .font(.body.weight(.medium))
+        .padding()
+        .multilineTextAlignment(.leading)
+        .frame(maxWidth: .infinity, alignment: alignment)
+        .foregroundColor(self.configuration.colors.foregroundColor)
+        .background {
+            RoundedRectangle(cornerRadius: 15, style: .continuous)
+                .foregroundColor(selected ? .black : .init(white: 0.8))
+        }
+    }
+
+    private var subscribeButton: some View {
+        PurchaseButton(
+            package: self.selectedPackage,
+            purchaseHandler: self.purchaseHandler,
+            colors: self.configuration.colors,
+            localization: self.selectedLocalization,
+            introEligibility: self.introEligibility[self.selectedPackage],
+            mode: self.configuration.mode
+        )
+    }
+
+    @ViewBuilder
+    private var backgroundImage: some View {
+        if let url = self.configuration.backgroundURL {
+            RemoteImage(url: url)
+                .blur(radius: 40)
+                .opacity(0.7)
+        } else {
+            DebugErrorView("Template configuration is missing background URL",
+                           releaseBehavior: .emptyView)
+        }
+    }
+
+    @ViewBuilder
+    private var iconImage: some View {
+        if let url = self.configuration.iconURL {
+            RemoteImage(url: url, aspectRatio: 1)
+                .clipShape(RoundedRectangle(cornerRadius: 10, style: .continuous))
+                .frame(maxWidth: 100)
```

Comment:
This is TBD @lburdock @guido732 (how big and rounded to display this icon). Please don't take my design like the source of truth or anything 😇